src/main/java/org/radarcns/phone/PhoneSensorProvider.java

```diff
@@ -26,12 +26,14 @@
 
 public class PhoneSensorProvider extends DeviceServiceProvider<PhoneState> {
     static final int PHONE_SENSOR_INTERVAL_DEFAULT = 200;
+    static final int PHONE_SENSOR_BATTERY_INTERVAL_DEFAULT = 60; // seconds
     static final String PHONE_SENSOR_INTERVAL = "phone_sensor_default_interval";
     static final String PHONE_SENSOR_GYROSCOPE_INTERVAL = "phone_sensor_gyroscope_interval";
     static final String PHONE_SENSOR_MAGNETIC_FIELD_INTERVAL = "phone_sensor_magneticfield_interval";
     static final String PHONE_SENSOR_STEP_COUNT_INTERVAL = "phone_sensor_steps_interval";
     static final String PHONE_SENSOR_ACCELERATION_INTERVAL = "phone_sensor_acceleration_interval";
     static final String PHONE_SENSOR_LIGHT_INTERVAL = "phone_sensor_light_interval";
+    static final String PHONE_SENSOR_BATTERY_INTERVAL = "phone_sensor_battery_interval";
```

Comment:
perhaps add `_seconds` to the setting name (same in README), since all other settings are milliseconds.